src/cmd/package.go

```diff
@@ -62,7 +62,20 @@ func newPackageCommand() *cobra.Command {
 	return cmd
 }
 
-type packageCreateOptions struct{}
+type packageCreateOptions struct {
+	confirm                 bool
+	output                  string
+	differentialPackagePath string
+	setVariables            map[string]string
```

Comment:
Not in scope for this PR, but good opportunity in the future to declare a type for `setVariables`'s `map[string]string`. e.g. `setVariables map[variable.Key]variable.Value` `setVariables variable.Variables` or another alternative